case/task/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Twitterized",
   "scripts": {
-      "start": "node app.js",
-      "watch": "gulp watch"
+      "start": "node app.js & npm run watch",
+      "watch": "./node_modules/.bin/gulp watch"
```

Comment:
npm scripts should [already](https://docs.npmjs.com/cli/run-script) be looking at this location:

> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts.

I think the only problem now is that I still kept `npm install -g gulp` in the readme, even though it is not actually needed.